lib/Cargo.toml

```diff
@@ -92,6 +91,7 @@ regex = "1.8.2"
 reqwest = { version = "0.11.18", default-features = false, features = ["json", "stream"], optional = true }
 roaring = { version = "0.10.1", features = ["serde"] }
 rocksdb = { version = "0.21.0", optional = true }
+rust_decimal = { git = "https://github.com/paupino/rust-decimal", rev = "714db9c4ccc83b742cb7613d12825dd82e845e8e", features = [ "maths" ] }
```

Comment:
Need a git dependency until a new version is released (with the `is_integer` change I [suggested](https://github.com/paupino/rust-decimal/issues/590))